### PR TITLE
#130 Create query to fetch former animal owner

### PR DESCRIPTION
**Pull request by mantasimkus**
Created GraphQL queries that fetch former animal owner(s). Wrote the respective integration tests. Solves issue #130.

### CODE REVIEW

File: src/sql-queries/formerAnimalOwner.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { QueryConfig } from 'pg';
+
+export const getFormerAnimalOwnersQuery = (): QueryConfig => {
+    const text = `
+        SELECT *
```
Comment (ignasposka) [inline src/sql-queries/formerAnimalOwner.ts:5]: This could fit into one line I think.

**Review by ignasposka — APPROVED** [review]
Good job! Please fix my comment and I will merge this PR.

**Review by AzisK — APPROVED** [review]
Good job

File: src/sql-queries/formerAnimalOwner.ts (hunk 1 above)
Comment (mantasimkus) [reply src/sql-queries/formerAnimalOwner.ts]: Done.

**Pull request merged**